src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -116,7 +186,7 @@ def get_reduce_params(node: gtir.FunCall) -> tuple[str, SymbolExpr, SymbolExpr]:
     return op_name, reduce_init, reduce_identity
 
 
-class LambdaToTasklet(eve.NodeVisitor):
+class LambdaToDataflow(eve.NodeVisitor):
```

Comment:
Just for my understanding and probably as additional comment.
Does this function builds everything inside a single map or multiple maps?